example/0000-buildconfig-nokvc.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: project.openshift.io/v1
+kind: Project
+metadata:
+  annotations:
+    openshift.io/description: ""
+    openshift.io/display-name: ""
+  labels:
+    kubernetes.io/metadata.name: silly
+  name: silly
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    app: silly-driver-container
+  name: silly-driver-container
+  namespace: silly
+spec: {}
+---
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  labels:
+    app: silly-driver-build
+  name: silly-driver-build
+  namespace: silly
+spec:
+  nodeSelector:
+    node-role.kubernetes.io/worker: ""
+  runPolicy: "Serial"
+  triggers:
+    - type: "ConfigChange"
+    - type: "ImageChange"
+  source:
+    dockerfile: |
+      #FROM quay.io/ebelarte/dtk-light:0.3
+      FROM image-registry.openshift-image-registry.svc:5000/dtk/dtk
+      WORKDIR /build/
+      RUN yum -y install git make sudo gcc \
```

Comment:
We already have those packages inside the DTK image.

Also, there is no yum repositories in the DTK image after the build. See [ref](https://bugzilla.redhat.com/show_bug.cgi?id=2070998)